src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_en.properties

```diff
@@ -525,6 +525,10 @@ rmdViewerTypeDescription = Where to display R Markdown documents when they have
 showPublishDiagnosticsTitle = Show diagnostic info when publishing
 showPublishDiagnosticsDescription = Whether to show verbose diagnostic information when publishing content.
 
+# Whether to show UI for publishing content to Posit Cloud.
+enableCloudPublishUiTitle = 
+enableCloudPublishUiDescription = Whether to show UI for publishing content to Posit Cloud.
```

Comment:
Not absolutely essential, but could also add these to https://github.com/rstudio/rstudio/blob/main/src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessorConstants_fr.properties.

A Google translate guestimate of the translation is this, but maybe check with @romainfrancois to make sure.

`S''il faut afficher l''interface utilisateur pour la publication de contenu sur Posit Cloud.`

(the double apostrophes are needed to render as a single apostrophe in these properties files)

Reply:
It is fine, but it looks from other properties we might want to use: 

``` 
Afficher ou non l''interface utilisateur·rice pour la publication de contenu sur Posit Cloud.
```

i.e. we seem to use the "<Verb> ou non" form, and "utilisateur·rice" is making "user" gender neutral and inclusive.